conformance/tests/basic/inferencepool_lifecycle.yaml

```diff
@@ -0,0 +1,95 @@
+# Basic InferencePool lifecycle test.
+# This manifest defines the minimal required fields to create valid
+# InferencePool and HTTPRoute resources, which the lifecycle test 
+# will use to verify that the controller recognizes and accepts the resource.
+
+# --- Minimal Backend Deployment (using agnhost echo server) ---
+# This Deployment provides Pods for the InferencePool to select.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: infra-backend-v1-deployment
```

Comment:
```suggestion
  name: infra-backend-v1
```

Reply:
inference_lifecycle removed per discussion in #795